### PR TITLE
rigaer78: convert to single file format

### PR DESCRIPTION
**Pull request by freifunkoperator**

### CODE REVIEW

File: host_vars/rigaer78-back-floor-2-kitchen/base.yml [hunk 1]
```diff
@@ -1,9 +0,0 @@
----
-
-location: rigaer78
-role: ap
-model: "avm_fritzbox-4040"
-
-# only useful in combination with port_untag patch that is not upstream
```
Comment (FFHener) [inline host_vars/rigaer78-back-floor-2-kitchen/base.yml:7]: Is this comment no longer valid?

File: locations/rigaer78.yml [hunk 2]
```diff
@@ -1,4 +1,97 @@
...
+  - hostname: rigaer78-zwingli
+    address: 10.31.119.132
+    snmp_profile: airos_8
+
+hosts:
+  - hostname: rigaer78-core
+    role: corerouter
+    model: "bananapi_bpi-r64"
+    wireless_profile: freifunk_default
+    dhcp_no_ping: false
```
Comment (FFHener) [inline locations/rigaer78.yml:22]: Why is this set to false as mgmt only contains static adresses?

**Comment by Noki** [comment]
In addition 3 devices are in outage or do not exist anymore:

rigaer78-back-floor-0-garage
rigaer78-back-floor-3-left
rigaer78-eap225

**Comment by pktpls** [comment]
LGTM, cleanups can happen another time

**Pull request merged**